packages/iot-app-kit-visualizations/src/components/charts/sc-webgl-base-chart/sc-webgl-base-chart.tsx

```diff
@@ -554,7 +556,10 @@ export class ScWebglBaseChart {
 
     const hasViewPortChanged = this.start.getTime() !== start.getTime() || this.end.getTime() !== end.getTime();
     if (hasViewPortChanged && !shouldBlockSetViewport) {
-      this.setViewport({ start, end, group: this.viewport.group }, 'chart-gesture');
+      const throttledSetViewport = throttle(() => {
```

Comment:
are you sure this works? every time this calls a new throttled function is created